Bug 1910140: fix the api dashboard with changes in upstream kube 1.20

tkashem opened the pull request:
- adjust graph titles 
- update the dashboard with changes in kube 1.20
- add new metrics to the dashboard

tkashem commented:
/retest

tkashem commented:
/retest

tkashem commented:
/retest

sttts commented:
/retest

sttts commented on manifests/0000_90_kube-apiserver-operator_05_api_performance_dashboard.yaml:
```diff
@@ -209,7 +209,7 @@ data:
           "timeRegions": [],
           "timeShift": null,
-          "title": "etcd Request Duration - 99th Quantile",
+          "title": "etcd Request Duration - 99th Percentile",
```
part of first commit?

tkashem replied:
@sttts this is part of the second commit - it's either 0.99 quantile or 99th percentile.

sttts commented:
Second commit seems to have many unrelated changes

tkashem commented:
@sttts in the second commit I did the following:
- upstream p&f changed the label values for the metrics, I had to fix it.
- there are a number of new metrics, I added these to the dashboard
- I added the metrics for `watch`.

sttts commented:
/lgtm
/approve

tkashem commented:
/retest
